### PR TITLE
feat: Added Polish translation [PT-186646535]

### PR DESCRIPTION
**Pull request by dougmartin**

### CODE REVIEW

**Comment by dougmartin** [comment]
Combined demo url (requires both SageModeler and the underlying building-models branch to load):

https://sagemodeler.concord.org/branch/update-langs-2023-12-11/app/?di=update-langs-2023-12-11&lang=pl

**Review by lublagg — APPROVED** [review]
LGTM 👍

File: src/code/views/simulation-run-panel-view.tsx [hunk 1]
```diff
@@ -44,7 +44,8 @@ export class SimulationRunPanelView extends Mixer<SimulationRunPanelViewProps, S
 
   private renderToggleButton() {
     const iconClass = this.state.simulationPanelExpanded ? "inspectorArrow-collapse" : "inspectorArrow-expand";
-    const doNotRotate = getDefaultLang() === "ko";
+    const defaultLang = getDefaultLang();
+    const doNotRotate = (defaultLang === "ko") || (defaultLang === "pl");
```
Comment (eireland) [inline src/code/views/simulation-run-panel-view.tsx:48]: Probably should add Greek to this list as well.
image
German is pushing it but not as bad.
image

**Review by eireland — APPROVED** [review]
Looks good. Though did make a suggestion re not rotating "Simulate" in Greek and German as well.

**Review by eireland — APPROVED** [review]
👍 LGTM

**Pull request merged**